protected/humhub/modules/search/controllers/MentioningController.php

```diff
@@ -33,37 +43,167 @@ public function getAccessRules()
         ];
     }
 
+    /**
+     * Find all users and spaces on mentioning request from RichText editor
+     *
+     * @return \yii\web\Response
+     */
     public function actionIndex()
     {
-        Yii::$app->response->format = 'json';
+        $keyword = (string)Yii::$app->request->get('keyword');
+
+        // Find users
+        $users = User::find()
+            ->visible()
+            ->search($keyword)
+            ->limit($this->module->mentioningSearchBoxResultLimit)
+            ->orderBy(['user.last_login' => SORT_DESC])
+            ->all();
 
         $results = [];
+        foreach ($users as $user) {
+            $results[] = $this->getUserResult($user);
+        }
+
+        $results = $this->appendMentioningSpaceResults($keyword, $results);
+
+        return $this->asJson($results);
+    }
+
+    /**
+     * Find space members on mentioning request from RichText editor on Post form
+     *
+     * @return \yii\web\Response
+     * @throws HttpException
+     */
+    public function actionPostSpaceMembers()
```

Comment:
Can we please rename to `actionSpace` so `/search/mentioning/space?id=4` .

Reply:
Ok, I changed, but please note I named firstly this as `PostSpaceMembers` in order to inform the action is used only for Post form side, because inside the action I added a check permission `$space->can(CreatePost::class)`, so it is not recommend to use this action from some other place in future.

Reply:
Ok, thank you for the hint. We can change the permission here in the future so that e.g. all those who have access to Space can search here.